docs/why-playwright.md

```diff
@@ -41,8 +42,16 @@ Playwright enables fast, reliable and capable automation across all modern brows
 
 * **TypeScript support**. Playwright ships with built-in types for auto-completion and other benefits.
 
-* **Debugging tools**. Playwright integrates with the [editor debugger](debug.md) to 
+* **Debugging tools**. Playwright works with the [editor debugger and browser developer tools](debug.md) to pause execution and inspect the web page.
 
 * **Language bindings**. Playwright is also available in [Python](https://github.com/microsoft/playwright-python) and [C#](https://github.com/hardkoded/playwright-sharp).
 
 * **Deploy tests to CI**. First-party [Docker image](docker/README.md) and [GitHub Actions](https://github.com/microsoft/playwright-github-action) to deploy tests to [your preferred CI/CD provider](ci.md).
+
+## Limitations
+
+* **Legacy Edge and IE11 support**. Playwright does not support legacy Microsoft Edge or IE11 ([deprecation notice](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-365-apps-say-farewell-to-internet-explorer-11-and/ba-p/1591666)). The new Microsoft Edge (on Chromium) is supported.
+
+* **Java language bindings**: The Playwright API cannot be used in Java or Ruby today. This is a temporary limitation as Playwright is built to support non-JavaScript language bindings.
```

Comment:
> ... to support non-JavaScript language bindings.

Maybe rephrase it as 'any language' I was confused a little bit first time I read.